e2e/.node-version

```diff
@@ -1 +1 @@
-18.16.0
+20.11.1
```

Comment:
Any reason not to use the current 20.x LTS version from April 10, 2024: `20.12.2`?

Reply:
Most of the other stuff uses 20.11.1? That's the only reason. It was a copy paste.